Data/Aeson/Types/Internal.hs

```diff
@@ -23,10 +23,14 @@ module Data.Aeson.Types.Internal
     -- * Type conversion
     , Parser
     , Result(..)
+    , JsonPathStep(..)
```

Comment:
I would like to see these named consistently with the rest of the package: `JSON`, not `Json`.

In fact, it would be even better to omit the `Json` or `JSON` prefix entirely as it's redundant.